app/views/podcasts/show.html.haml

```diff
@@ -47,57 +47,19 @@
       - if @episodes[:new].length == 0
         No new episodes
       - else
-        -@episodes[:new].each do |episode|
-          .media.episode
-            .media-left
-              = link_to episode do
-                %img{ src: asset_path("play-button.jpg") }
-            .media-body
-              %h5.media-heading
-                =link_to episode.title, episode
-              .media-text
-                = episode.summary.truncate_words 20
-              .date-inline
-                = episode.published_at
-            .media-right
-              = episode.published_at
+        = render partial: "episode", locals: { episodes: @episodes[:new] }
 
       .divider
         Played Episodes
         %hr
       - if @episodes[:old].length == 0
         No new episodes
       - else
-        -@episodes[:old].each do |episode|
-          .media.episode
-            .media-left
-              = link_to episode do
-                %img{ src: asset_path("play-button.jpg") }
-            .media-body
-              %h5.media-heading
-                =link_to episode.title, episode
-              .media-text
-                = episode.summary
-              .date-inline
-                = episode.published_at
-            .media-right
-              = episode.published_at
+        = render partial: "episode", locals: { episodes: @episodes[:old] }
 
     - else
       -@episodes.each do |episode|
-        .media.episode
-          .media-left
-            = link_to episode do
-              %img{ src: asset_path("play-button.jpg") }
-          .media-body
-            %h5.media-heading
-              =link_to episode.title, episode
-            .media-text
-              = episode.summary
-            .date-inline
-              = episode.published_at
-          .media-right
-            = episode.published_at
+        = render partial: "episode", locals: { episodes: @episodes }
```

Comment:
Now the partial should be called "episodes" with an "S" at the end, since it is rendering a list